send-email/send-email.py

```diff
@@ -0,0 +1,48 @@
+from azure.communication.email import EmailClient, EmailContent,EmailAddress,EmailMessage,EmailRecipients
+import time
+def main():
+    try:
+        connection_string = "<ACS_CONNECTION_STRING>"
+        client = EmailClient.from_connection_string(connection_string)
+        sender = "<SENDER_EMAIL>"
+        content = EmailContent(
+            subject="Test email from Python",
+            plain_text="This is plaintext body of test email.",
+            html= "<html><h1>This is the html body of test email.</h1></html>",
+        )
+
+        recipient = EmailAddress(email="<RECIPIENT_EMAIL>", display_name="<RECIPIENT_DISPLAY_NAME>")
+
+        message = EmailMessage(
+            sender=sender,
+            content=content,
+            recipients=EmailRecipients(to=[recipient])
+        )
+
+        response = client.send(message)
+        if (response == None or response.message_id=='undfined' or response.message_id==''):
```

Comment:
"undefined" is misspelled here

Reply:
good catch, updated with more formatting fixes.